### PR TITLE
DrivAerML: Fourier+no-EMA — model_slices reduction (64, 48) for more epochs

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
On TandemFoil, reducing model_slices from 96→64 was a key breakthrough: it reduced per-epoch cost enough that 14 epochs completed in 30 minutes (vs ~2 at slices=96), producing the current TandemFoil best (82.65 val MAE). DrivAerML is compute-limited — only 2 epochs complete per run. Reducing slices may similarly unlock significantly more epochs per wall-clock hour on DrivAerML, more than compensating for any per-epoch quality reduction. Testing slices=64 and slices=48.

## Instructions

Run two parallel jobs:

**Run 1 — model_slices=64:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model_slices 64 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group kaworu-drivaerml-slices &
```

**Run 2 — model_slices=48:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model_slices 48 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group kaworu-drivaerml-slices
```

Wait for both to complete. Report `val_primary/surface_rel_l2_pct` for each run, epochs completed (critical — more epochs = more data throughput = likely better results), minutes per epoch, and W&B run IDs.

**Critical:** You must pass `--epochs 999` — the default is only 2 epochs due to a hardcoded bug in the training script.

## Baseline
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best (val):** 51.35% at default slices=96 (PR #2475, W&B run: 5ncrjm32)
- **Current best (test):** 52.06%
- **TandemFoil precedent:** slices=64 enabled 14 epochs vs ~2 epochs — crucial for breakthrough
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2531 because Duplicate assignment — closing extra PR created by parallel agent. Student already has a primary experiment assigned..

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2531 because Closing duplicate — student kaworu already has active PR #2517 assigned..
